server/src/main/java/org/dst/server/service/DstListServiceImpl.java

```diff
@@ -34,9 +37,93 @@ public DstListServiceImpl(KVStore kvStore) {
             ListProtocol.ListGetResponse.newBuilder();
 
     List<String> values = store.list().get(request.getKey());
-    values.forEach(value -> responseBuilder.addValues(value));
+    // TODO change protocol
+    Optional.ofNullable(values)
+            .ifPresent(u -> {
```

Comment:
`u` is not meaningful, please rename it.

Reply:
got it!